diff -ur ../compute-image-packages-2582dfef02e559d7accb7200987910950e74b747.orig/google-daemon/usr/lib/systemd/system/google-accounts-manager.service ./google-daemon/usr/lib/systemd/system/google-accounts-manager.service
--- ../compute-image-packages-2582dfef02e559d7accb7200987910950e74b747.orig/google-daemon/usr/lib/systemd/system/google-accounts-manager.service	2014-05-08 18:24:13.000000000 +0000
+++ ./google-daemon/usr/lib/systemd/system/google-accounts-manager.service	2014-06-19 07:51:12.523767527 +0000
@@ -6,7 +6,7 @@
 
 [Service]
 Type=simple
-ExecStart=/usr/share/google/google_daemon/manage_accounts.py
+ExecStart=/usr/share/google_daemon/manage_accounts.py
 
 [Install]
 WantedBy=multi-user.target
diff -ur ../compute-image-packages-2582dfef02e559d7accb7200987910950e74b747.orig/google-daemon/usr/lib/systemd/system/google-address-manager.service ./google-daemon/usr/lib/systemd/system/google-address-manager.service
--- ../compute-image-packages-2582dfef02e559d7accb7200987910950e74b747.orig/google-daemon/usr/lib/systemd/system/google-address-manager.service	2014-05-08 18:24:13.000000000 +0000
+++ ./google-daemon/usr/lib/systemd/system/google-address-manager.service	2014-06-19 07:51:27.069765987 +0000
@@ -5,7 +5,7 @@
 
 [Service]
 Type=simple
-ExecStart=/usr/share/google/google_daemon/manage_addresses.py
+ExecStart=/usr/share/google_daemon/manage_addresses.py
 
 [Install]
 WantedBy=multi-user.target
